sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -281,7 +281,7 @@
       "post": {
         "summary": "Create, update, or delete the identities of multiple devices from the IoT hub identity registry.",
         "description": "Create, update, or delete the identiies of multiple devices from the IoT hub identity registry. A device identity can be specified only once in the list. Different operations (create, update, delete) on different devices are allowed. A maximum of 100 devices can be specified per invocation. For large scale operations, consider using the import feature using blob storage(https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities).",
-        "operationId": "RegistryManager_BulkDeviceCRUD",
+        "operationId": "Devices_BulkDeviceCRUD",
```

Comment:
"Devices_BulkRegistryOperations" instead, since CRUD now applies to modules, too

Reply:
Yes, David had a same comment and I also couldn't come up with better suggestion than Operations. I'll go ahead change that then